src/Pos/Block/Network/Listeners.hs

```diff
@@ -78,8 +78,8 @@ handleBlockHeaders
     :: forall ssc m.
        (SscWorkersClass ssc, WorkMode ssc m)
     => (ListenerSpec m, OutSpecs)
-handleBlockHeaders = listenerConv $ \__ourVerInfo nodeId conv -> do
+handleBlockHeaders = listenerConv @Void $ \__ourVerInfo nodeId conv -> do
```

Comment:
`Void` means that no messages will be actually send (there's no way to create a value of `Void`). It doesn't matter what the other side expects because it won't receive anything.